pkg/sources/fake/fake.go

```diff
@@ -6,15 +6,36 @@ import (
 
 	"telemetry/pkg/msgs"
 	"telemetry/pkg/pubsub"
+	"telemetry/pkg/util/randutil"
 )
 
 const (
 	// interval between fake data readings
-	readInterval = 20 * time.Millisecond
+	readInterval = 200 * time.Millisecond
+
+	// TODO(karlding): change these to the actual values
+	rightTurnOn     = 1
```

Comment:
Also here is a good case to use `iota` for now rather than manually defining the numbers. At least until we have definite values.

https://golang.org/ref/spec#Iota

```go
const (
  rightTurnOn   = iota // 0
  rightTurnOff         // 1
  ...
)
```